app/src/main/java/com/example/hcm_102_0006/android_project_m/view/adapter/AdapterShowGenresDetail.java

```diff
@@ -0,0 +1,75 @@
+package com.example.hcm_102_0006.android_project_m.view.adapter;
+
+import android.content.Context;
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.example.hcm_102_0006.android_project_m.R;
+import com.example.hcm_102_0006.android_project_m.data.model.Genres;
+import com.example.hcm_102_0006.android_project_m.databinding.ItemGenreDetailBinding;
+import com.example.hcm_102_0006.android_project_m.ui.moviedetail.MovieDetailViewModel;
+
+import java.util.List;
+
+/**
+ * Created by hcm-102-0006 on 29/11/2017.
+ */
+
+public class AdapterShowGenresDetail extends RecyclerView.Adapter<AdapterShowGenresDetail.GenresViewHolder> {
+    private List<Genres> mGenres;
+    private Context mContext;
+    public static final String BUNDLE_GENRES_DETAIL = "BUNDLE_GENRES_DETAIL";
+    public static final int KEY_GENRES_DETAIL = 123;
```

Comment:
2.8 Class member ordering

This is not the only correct solution, but it is recommended to use the following order :

Constants
Fields
Constructors
Override methods and callbacks (public or private)
Public methods
Private methods
Inner classes or interfaces